### ISSUE TITLE
Page layouts provided in svelte ui package should provide consistent aria-labels

### ISSUE THREAD

**Issue by feloy** [containers/podman-desktop#7740]
### Is your enhancement related to a problem? Please describe

Merging a page from NavPage to DetailsPage, some unit tests are failing and cannot be adapted as there are some aria-labels defined in NavPage not defined in DetailsPage

### Describe the solution you'd like

Consistent aria-labels across different pages layouts (NavPage, DetailsPage, FormPage)
